core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringRollbackTest.java

```diff
@@ -61,7 +66,12 @@ public void testSplitBeforeRollback() throws Exception {
       SplitMode.BOTH_DEGRADED.split(this);
 
       transactionManager.resume(transaction);
-      transaction.runCommit(true);
+      try {
+         transaction.runCommit(true);
+         fail("RollbackException expected!");
```

Comment:
I would have liked a more specific error message, or a comment on why the commit fails.